src/server/replica.cc

```diff
@@ -1229,7 +1229,8 @@ Replica::Info Replica::GetInfo() const {
     res.host = master_context_.host;
     res.port = master_context_.port;
     res.master_link_established = (state_mask_.load() & R_TCP_CONNECTED);
-    res.sync_in_progress = (state_mask_.load() & R_SYNCING);
+    res.full_sync_in_progress = (state_mask_.load() & R_SYNCING);
+    res.full_sync_done = (state_mask_.load() & R_SYNC_OK);
```

Comment:
I am not very familiar with those state flags so trust it translates to the semantics of those member variables.